force-app/main/default/classes/RD2_ERecurringDonationsSelector.cls

```diff
@@ -75,7 +75,8 @@ public inherited sharing class RD2_ERecurringDonationsSelector {
                                 + 'npe03__Date_Established__c, '
                                 + 'npe03__Next_Payment_Date__c, '
                                 + 'npe03__Total_Paid_Installments__c, '
-                                + 'Lastmodifieddate '
+                                + 'Lastmodifieddate ,'
+                                + '(SELECT Active__c, Campaign__c, Campaign__r.Name, StartDate__c, EndDate__c, DayOfMonth__c, InstallmentAmount__c, InstallmentFrequency__c, InstallmentPeriod__c, PaymentMethod__c, RecurringDonation__c, IsPause__c, StatusReason__c FROM RecurringDonationSchedules__r) '
```

Comment:
Continuing this conversation from the original [PR thread](https://github.com/SalesforceFoundation/NPSP/pull/7020#discussion_r914173740).

Reply:
Because the object is protected, an admin will never be able to assign any CRUD or FLS to the object or its fields because they cannot see the object anywhere in their org. It's like it doesn't exist (to them). 

It is in a MasterDetail relationship as you note, so technically if the user has READ on the RD object they will have READ on the RDSchedule object too. What may break this though is the use of `WITH_SECURITY_ENFORCED` on the query. That will enforce field perms, but it will never be possible for an Admin to grant field visibility to RDSchedule fields because they cannot see the object to do so. I'm almost positive we saw this elsewhere. I am nearly sure this will fail in a package context.

Reply:
Ok, I get it. I'll work on this.